README.markdown

```diff
@@ -160,7 +160,7 @@ indicates how often that measurement was repeated.
 The chart on the right contains the raw measurements from which the
 kernel density estimate was built. The $x$ axis indicates the number
 of loop iterations, while the $y$ axis shows measured execution time
-for the given number of iterations. The line "behind" the values is a
+for the given number of iterations. The line “behind” the values is a
```

Comment:
Genuinely curious: what is the rationale for using this style of double quotes?

Reply:
FTR it's a separate commit and we can revert it if you want. 

Tldr; It's typography 101. A longer answer from the top of Google search: https://practicaltypography.com/straight-and-curly-quotes.html (fun fact: being in Indiana, you might have heard of the author, a prominent Racketeer at one point)